examples/lighting-app/silabs/src/LightingManager.cpp

```diff
@@ -307,3 +363,72 @@ void LightingManager::OnTriggerOffWithEffect(OnOffEffect * effect)
     LightMgr().mOffEffectArmed = true;
     LightMgr().StartTimer(offEffectDuration);
 }
+
+#if (defined(SL_MATTER_RGB_LED_ENABLED) && SL_MATTER_RGB_LED_ENABLED == 1)
+bool LightingManager::InitiateLightctrlAction(int32_t aActor, Action_t aAction, uint16_t size, uint32_t aAttributeId,
```

Comment:
You don't use `uint16_t size` in this function. You can drop the parameter

Its probably easier and cleaner to have 1 function for each action rather than a case and multiple ifs.
At that point you wouldn't need aAttributeId parameter either

Reply:
Removed size param from function. 
Rather than having 3 functions , I have created single function to handle all color formats lightctrlAction function by passing Attribute ID .